server/go/README.md

```diff
@@ -0,0 +1,48 @@
+# Stripe Payments Demo - Go Server
+
+This demo uses a simple [Echo](https://echo.labstack.com/) application as the server.
+
+## Payments Integration
+
+- [`app.go`](app.go) contains the routes that interface with Stripe to create 
+
+## Requirements
+
+You’ll need the following:
+
+- [Go 1.11 or later](https://golang.org/doc/install) (for module support)
+- Modern browser that supports ES6 (Chrome to see the Payment Request, and Safari to see Apple Pay).
+- Stripe account to accept payments ([sign up](https://dashboard.stripe.com/register) for free!)
+
+## Getting Started
+
+Copy the example environment variables file `.env.example` from the root of the repo into your own environment file called `.env`:
+
+```
+cp .env.example .env
+```
+
+
+Run the application from this directory
```

Comment:
can you include the `cd server/go/` command, to make sure folks understand the context of "this directory"?